src/utils/luisAppSetup.ts

```diff
@@ -1,42 +1,42 @@
 import { LuisRecognizer } from 'botbuilder-ai';
 
-export const LUISUnblockSetup = (stepContext:any) => {
```

Comment:
as you rename the function name, please remove one reference for the old file in unblockDialogs/unblockLookup.ts